docs/WritingTests.asciidoc

```diff
@@ -160,6 +160,34 @@ Variables are accessible via the *get_var* and *check_var* functions.
 
 == Test Development tricks
 
+=== Capturing kernel exceptions and/or any other exceptions from the serial console
+
+The test developer can define if he wants to raise a soft or a hard failure when a pattern 
```

Comment:
Rewritten as `Soft and hard failures can be triggered on demand by regular expressions when they match the serial output which is done...` could sound better and less redundant